share/lib/python/neuron/rxd/rxd.py

```diff
@@ -1896,7 +1896,7 @@ def _init():
     _setup_matrices()
     # if species._has_1d and species._1d_submatrix_n():
     # volumes = node._get_data()[0]
-    # zero_volume_indices = (numpy.where(volumes == 0)[0]).astype(numpy.int_)
+    # zero_volume_indices = (numpy.where(volumes == 0)[0]).astype(ctypes.c_long)
```

Comment:
Similar to other comment: I changed it here out of paranoia that this code ever gets used again.